bundle-workflow/tests/tests_assemble_workflow/test_bundle_recorder.py

```diff
@@ -87,3 +87,60 @@ def test_write_manifest(self):
             data = self.bundle_recorder.get_manifest().to_dict()
             with open(manifest_path) as f:
                 self.assertEqual(yaml.safe_load(f), data)
+
+    def test_record_component_public(self):
+        self.bundle_recorder.public_url = 'https://ci.opensearch.org/ci/xyz'
```

Comment:
Why is there a `xyz` or `dbc` prefix before meeting `builds` folder?

Reply:
Its a non-sense identifier to which jenkins cluster actually has the data, see https://github.com/opensearch-project/opensearch-build/tree/main/deployment#overall-system

Reply:
It feels like it should be `opensearch` or `openseach-dashboards` instead of `dbc`, possible?

Reply:
It feels like it should be `opensearch` or `opensearch-dashboards` rather than `dbc`, possible?

Reply:
Renamed, but we still need to update the schema but that will not require changes into this code as it should happen at the jenkinsfile level